pdns/common_startup.cc

```diff
@@ -207,6 +207,8 @@ void declareArguments()
   ::arg().set("default-zsk-algorithm","Default ZSK algorithm")="";
   ::arg().set("default-zsk-size","Default ZSK size (0 means default)")="0";
   ::arg().set("max-nsec3-iterations","Limit the number of NSEC3 hash iterations")="500"; // RFC5155 10.3
+  ::arg().set("default-publish-cdnskey","Default value for PUBLISH-CDNSKEY")="";
```

Comment:
Perhaps this should be a switch? It is only an integer in the metadata because other metadata uses 0/1 for bools.

```suggestion
  ::arg().setSwitch("default-publish-cdnskey","Set PUBLISH-CDNSKEY to 1 for all zones without this metadata item");
```

Reply:
It's not a switch, it has three potential values (0/1/empty).